jenkins/plots/index.html

```diff
@@ -28,15 +28,15 @@ <h2 id="stress-perf-build-status">Stress Perf <a href="http://build.wiredtiger.c
 <h2 id="ycsb-mongo-master-wt-developbuild-status">YCSB Mongo Master + WT Develop<a href="http://build.wiredtiger.com:8080/job/mongodb-perf-ycsb-develop/"><img src="http://build.wiredtiger.com:8080/buildStatus/icon?job=mongodb-perf-ycsb-develop" alt="Build Status" /></a></h2>
 <p><a href="http://build.wiredtiger.com:8080/job/mongodb-perf-ycsb-develop/plot"><img src="http://build.wiredtiger.com:8080/job/mongodb-perf-ycsb-develop/plot/getPlot?index=0&amp;width=750&amp;height=450" /></a> <a href="http://build.wiredtiger.com:8080/job/mongodb-perf-ycsb-develop/plot"><img src="http://build.wiredtiger.com:8080/job/mongodb-perf-ycsb-develop/plot/getPlot?index=1&amp;width=750&amp;height=450" /></a> <a href="http://build.wiredtiger.com:8080/job/mongodb-perf-ycsb-develop/plot"><img src="http://build.wiredtiger.com:8080/job/mongodb-perf-ycsb-develop/plot/getPlot?index=2&amp;width=750&amp;height=450" /></a> <a href="http://build.wiredtiger.com:8080/job/mongodb-perf-ycsb-develop/plot"><img src="http://build.wiredtiger.com:8080/job/mongodb-perf-ycsb-develop/plot/getPlot?index=3&amp;width=750&amp;height=450" /></a> <a href="http://build.wiredtiger.com:8080/job/mongodb-perf-ycsb-develop/plot"><img src="http://build.wiredtiger.com:8080/job/mongodb-perf-ycsb-develop/plot/getPlot?index=4&amp;width=750&amp;height=450" /></a> <a href="http://build.wiredtiger.com:8080/job/mongodb-perf-ycsb-develop/plot"><img src="http://build.wiredtiger.com:8080/job/mongodb-perf-ycsb-develop/plot/getPlot?index=5&amp;width=750&amp;height=450" /></a> <a href="http://build.wiredtiger.com:8080/job/mongodb-perf-ycsb-develop/plot"><img src="http://build.wiredtiger.com:8080/job/mongodb-perf-ycsb-develop/plot/getPlot?index=6&amp;width=750&amp;height=450" /></a> <a href="http://build.wiredtiger.com:8080/job/mongodb-perf-ycsb-develop/plot"><img src="http://build.wiredtiger.com:8080/job/mongodb-perf-ycsb-develop/plot/getPlot?index=7&amp;width=750&amp;height=450" /></a> <a href="http://build.wiredtiger.com:8080/job/mongodb-perf-ycsb-develop/plot"><img src="http://build.wiredtiger.com:8080/job/mongodb-perf-ycsb-develop/plot/getPlot?index=8&amp;width=750&amp;height=450" /></a></p>
-<h2 id="ycsb-mongo-master-wt-mongodb-3.0-build-status">YCSB Mongo Master + WT mongodb-3.0 <a href="http://build.wiredtiger.com:8080/job/mongodb-perf-ycsb-mongodb-3.0/"><img src="http://build.wiredtiger.com:8080/buildStatus/icon?job=mongodb-perf-ycsb-mongodb-3.0" alt="Build Status" /></a></h2>
+<h2 id="ycsb-mongo-master-wt-mongodb-3.0-build-status">YCSB Mongo Master + WT mongodb-3.0 <a href="http://build.wiredtiger.com:8080/job/mongodb-perf-ycsb-mongodb-3.0/"><embed src="http://build.wiredtiger.com:8080/buildStatus/icon?job=mongodb-perf-ycsb-mongodb-3.0" /></a></h2>
```

Comment:
I didnt notice that one. It gets auto generated. 
Maybe different version of pandoc that gets used for auto generation.